airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -270,6 +291,26 @@ private static boolean isCdc(JsonNode config) {
     return !(config.get("replication_slot") == null);
   }
 
+  private static AirbyteStream removeIncrementalWithoutPk(AirbyteStream stream) {
```

Comment:
can you add a comment explaining why we do this? i think the answer is CDC doesn't support tables without PKs, but it's not super obvious just reading the code.